components/raftstore/src/store/worker/pd.rs

```diff
@@ -1096,23 +1093,12 @@ where
                 spawn_local(f);
             }
 
-            Task::Heartbeat {
-                term,
-                region,
-                peer,
-                down_peers,
-                pending_peers,
-                written_bytes,
-                written_keys,
-                approximate_size,
-                approximate_keys,
-                replication_status,
-            } => {
-                let approximate_size = approximate_size.unwrap_or_else(|| {
-                    get_region_approximate_size(&self.db, &region, 0).unwrap_or_default()
+            Task::Heartbeat(hb_task) => {
+                let approximate_size = hb_task.approximate_size.unwrap_or_else(|| {
+                    get_region_approximate_size(&self.db, &hb_task.region, 0).unwrap_or_default()
                 });
-                let approximate_keys = approximate_keys.unwrap_or_else(|| {
-                    get_region_approximate_keys(&self.db, &region, 0).unwrap_or_default()
+                let approximate_keys = hb_task.approximate_keys.unwrap_or_else(|| {
+                    get_region_approximate_keys(&self.db, &hb_task.region, 0).unwrap_or_default()
```

Comment:
`get_region_approximate_size` is unnecessary here. We do not need to read rocksdb here,

Reply:
Please remove it.

Reply:
Done.